### PR TITLE
Make reset_pulp function compatible with RHEL 6

### PR DESCRIPTION
**Pull request by Ichimonji10**
The `reset_pulp` function executes `runuser -u apache pulp-manage-db`.
This doesn't work on RHEL 6, because RHEL 6 ships with an old version of
runuser that doesn't support `-u`. Work around this limitation.

For exact details, see runuser(1).

### CODE REVIEW

**Comment by Ichimonji10** [comment]
I've tested this on a pair of Fedora 23 systems. Can someone test this on RHEL 6?

@elyezer @seandst

File: pulp_smash/utils.py [hunk 1]
```diff
@@ -100,10 +100,20 @@ def reset_pulp(server_config):
...
+    #
+    # Why not use runuser's `-u` flag? Because RHEL 6 ships an old version of
+    # runuser that doesn't support the flag, and RHEL 6 is a supported Pulp
+    # platform.
     client = cli.Client(server_config)
     prefix = '' if is_root(server_config) else 'sudo '
     client.run('mongo pulp_database --eval db.dropDatabase()'.split())
-    client.run((prefix + 'runuser -u apache pulp-manage-db').split())
+    client.run((
+        prefix + 'runuser --shell /usr/bin/sh apache --command pulp-manage-db'
```
Comment (elyezer) [inline pulp_smash/utils.py]: On RHEL6:

```console
$ sudo runuser --shell /usr/bin/sh apache --command pulp-manage-db
runuser: /usr/bin/sh: No such file or directory
```

Comment (seandst) [inline pulp_smash/utils.py]: /usr/bin/sh is correct for newer distros, but they provide backward compatibility with a symlink of /bin to /usr/bin, so /bin/sh is the easy compromise if not the correct choice.

Comment (Ichimonji10) [inline pulp_smash/utils.py]: Will update.

**Comment by Ichimonji10** [comment]
Updated.

**Pull request merged**